pdf-toolbox/src/test/java/com/lowagie/examples/objects/bookmarks/PageLabels.java

```diff
@@ -65,5 +66,19 @@ public static void main(String[] args) {
 
         // step 5: we close the document
         document.close();
+
+        try (PdfReader reader = new PdfReader("PageLabels.pdf")) {
```

Comment:
AFAIU this code logs generated page label names. Why is it necessary? This is not a test, more like debug code which should be removed at production. Am I right? Please, explain

Reply:
As I changed some deep code, I prefer having some code that uses the changed code. To be sure not to have broken anything.

I'll transform this an assert as it is a test class.